### ISSUE TITLE
Acapella Downloader detected as trojan

### ISSUE THREAD

**Issue by threetwooneElol12** [weespin/WillFromAfarDownloader#19]
it detects 'AcapellaDownloader' as trojan when my webroot sent me notification :/

1. go to downloads
2. Double Click on 'Release'
3. find 'AcapellaDownloader' when extracted
4. if so, your virus could detect the downloader as a trojan, and i cant fix it

 - OS: Windows 11
 - Edition: Home
 - Installed on: 10/5/21
 - Experience: Windows Feature Experience Pack 1000.22000.348.0
 - Version: 21H2
- Build: 22000.348

idk why i put the entire PC info

nothing else needed
idk what issue is but hopefully this will be sent to who i want it to be sent to

**Comment by threetwooneElol12** [comment]
i hate what i just did

**Comment by weespin** [comment]
[Welcome to the world of unsigned open sourced binaries](https://www.pandasecurity.com/en/mediacenter/security/false-positives-what-are-they/)
tl;dr just get a better antivirus or use a virustotal 🤦
